design/secrets.md

```diff
@@ -0,0 +1,109 @@
+# Design proposal template (replace with your proposal's title)
+
+Currently, Velero only supports a single AIM access credential secret per location provider/plugin. Velero creates and stores the plugin credential secret under the hard-coded key `secret.cloud-credentials.data.cloud`.
```

Comment:
Typo, IAM.

Reply:
The plugin's initContainer just copies the plugin into the /plugins directory and doesn't actually run the plugin in the container.  The plugin is run in the main container as a process.

Reply:
Thanks for the explanation.